app/models/parser/twitter_profile.rb

```diff
@@ -19,25 +19,38 @@ def patterns
 
     # Main function for class
     def parse_data_for_parser(doc, _original_url, _jsonld_array)
-      @url = replace_subdomain_pattern(url)
-      username = url.match(/^https?:\/\/(www\.)?twitter\.com\/([^\/]+)$/)[2]
+      @url.gsub!(/\s/, '')
```

Comment:
**question** are we seeing whitespace in the URL here? curious what this is needed for. same question for in twitter item

Reply:
we had a test that checked if any whitespaces were removed. So I assumed this was something we saw before and wanted to address.